CMakeLists.txt

```diff
@@ -42,18 +49,6 @@ find_package(FFMPEG 3.4 REQUIRED
   swresample
 )
 
-find_package(frei0r)
```

Comment:
Remove the find module too:
https://github.com/olive-editor/olive/blob/master/cmake/Findfrei0r.cmake

Reply:
File removed in commit 4f54fb1